charms/layers/java/reactive/java.py

```diff
@@ -14,14 +14,15 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 # pylint: disable=c0111,c0103,c0301
+import openjdk
```

Comment:
> Imports should be grouped in the following order:
> 1. standard library imports
> 2. related third party imports
> 2. local application/library specific imports

https://www.python.org/dev/peps/pep-0008/#imports


It's best to use a linter to avoid such mistakes. I use pylint.